clients/algoliasearch-client-swift/Sources/Core/Common/AlgoliaError.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  File.swift
+//
+//
+//  Created by Thomas Raffray on 22/12/2023.
```

Comment:
```suggestion
//  Created by Algolia on 22/12/2023.
```
also `File.swift` is not the correct name